solvis_graphql_api/composite_solution/filter_set_logic_options.py

```diff
@@ -1,12 +1,17 @@
 """The API schema for FilterSetLogicOptions."""
+from typing import Any, Tuple
 
 import graphene
+from solvis.inversion_solution import typing
```

Comment:
@DrCuriosity  suggest:

`import solvis.inversion_solution.typing` as its confusing (for me at least) to see this referred to as just  typing`. Even thought it's a tiny module, when looking at it in a github PR it still cacught me out.

Reply:
That's fair - I definitely shouldn't be overriding a name that's part of the Python Standard Library, especially as we're _also_ importing functions from that `typing`. Switching to the more verbose input.

(In passing, also noticed our `if TYPE_CHECKING:` block in cached.py wasn't in code coverage. Since it's only there for mypy purposes, I've added that to the `exclude_lines` list in setup.cfg.)